pyomo/contrib/mindtpy/tests/test_mindtpy_feas_pump.py

```diff
@@ -0,0 +1,292 @@
+            log_infeasible_constraints(model)
+            self.assertTrue(is_feasible(model, self.get_config(opt)))
+
+    def test_FP_8PP_Norm_infinity(self):
+        """Test the feasibility pump algorithm."""
+        with SolverFactory('mindtpy') as opt:
+            model = EightProcessFlowsheet()
+            print(
+                '\n Solving 8PP problem using feasibility pump with Norm infinity in mip projection problem')
+            results = opt.solve(model, strategy='feas_pump',
+                                mip_solver=required_solvers[1],
+                                nlp_solver=required_solvers[0],
+                                bound_tolerance=1E-5,
+                                fp_master_norm='L_infinity')
+            log_infeasible_constraints(model)
+            self.assertTrue(is_feasible(model, self.get_config(opt)))
+
+    def test_FP_8PP_Norm_infinity_with_norm_constraint(self):
+        """Test the feasibility pump algorithm."""
+        with SolverFactory('mindtpy') as opt:
+            model = EightProcessFlowsheet()
+            print(
+                '\n Solving 8PP problem using feasibility pump with Norm infinity in mip projection problem')
+            results = opt.solve(model, strategy='feas_pump',
+                                mip_solver=required_solvers[1],
+                                nlp_solver=required_solvers[0],
+                                bound_tolerance=1E-5,
+                                fp_master_norm='L_infinity',
+                                fp_norm_constraint=False)
+            log_infeasible_constraints(model)
+            self.assertTrue(is_feasible(model, self.get_config(opt)))
+
+    def test_FP_simpleMINLP(self):
+        """Test the feasibility pump algorithm."""
+        with SolverFactory('mindtpy') as opt:
+            model = SimpleMINLP()
+            print('\n Solving 8PP problem using feasibility pump')
+            results = opt.solve(model, strategy='feas_pump',
+                                mip_solver=required_solvers[1],
+                                nlp_solver=required_solvers[0],
+                                bound_tolerance=1E-5)
+            log_infeasible_constraints(model)
+            self.assertTrue(is_feasible(model, self.get_config(opt)))
+
+    def test_FP_Feasibility_Pump1(self):
+        """Test the feasibility pump algorithm."""
+        with SolverFactory('mindtpy') as opt:
+            model = Feasibility_Pump1()
+            print('\n Solving Feasibility_Pump1 with feasibility pump')
+            results = opt.solve(model, strategy='feas_pump',
+                                mip_solver=required_solvers[1],
+                                nlp_solver=required_solvers[0],
+                                bound_tolerance=1E-5)
+            log_infeasible_constraints(model)
+            self.assertTrue(is_feasible(model, self.get_config(opt)))
+
+    # def test_FP_Feasibility_Pump2(self):
```

Comment:
Is this still an issue? Why would it be a problem? If the fixed problem is an LP, so what? IPOPT should be able to solve that LP.

Reply:
Not really.

Reply:
Then uncomment it.